types/firestore.d.ts

```diff
@@ -1627,7 +1670,7 @@ declare namespace FirebaseFirestore {
      * of the query's order by.
      * @return The created Query.
      */
-    startAt(...fieldValues: any[]): Query<T>;
+    startAt(...fieldValues: unknown[]): Query<AppModelType, DbModelType>;
```

Comment:
Please undo the change from `any` to `unknown` unless it is strictly required for this PR.